### PR TITLE
move docs link

### LINKED ISSUE #680: All about that Docs button!
Been wondering about the docs button.

- Does it link to the internet hosted docs? Or a version hosted by the server you're on? 🤔 
- Should probably like to the relevant page?
- Does it have to be so big and dominant?

Prompted to make this issue also because I had an idea:

image

- Replace the 'back arrow' which 1) isn't useful 2) has really weird back behaviour 3) maybe wasn't intended as a button but it looks (to me!) to be one with an `info` button for the current page. There's still issues with it, but when isn't there? :)

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #680 

moved docs to the bottom section of the app drawer, touching as many files as possible for such a simple change 🤷

### CODE REVIEW

File: packages/common/src/ui/components/navigation/ExternalNavLink.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import React, { FC } from 'react';
```
Comment (mark-prins) [inline packages/common/src/ui/components/navigation/ExternalNavLink.tsx:1]: all this to have a nav link that looks the same as the others, but doesn't link to a relative path, or both with selected state

File: packages/host/src/components/AppBar/SectionIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  return undefined;
+};
+
+export const SectionIcon: React.FC = () => {
+  const t = useTranslation('app');
+  const section = getSection();
+  const icon = getIcon(section);
+
+  return icon ? (
+    <Tooltip title={t(section as LocaleKey)}>
```
Comment (mark-prins) [inline packages/host/src/components/AppBar/SectionIcon.tsx:46]: iffy, and probably not required? not sure that we need a tooltip on the icon.. generally think that it is helpful tho

File: packages/host/src/components/AppBar/SectionIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    const match = matchPath(
+      RouteBuilder.create(route ?? '')
+        .addWildCard()
+        .build(),
+      location.pathname
+    );
+    if (!!match)
+      return {
+        icon: getIcon(route),
+        titleKey: route as LocaleKey,
```
Comment (mark-prins) [inline packages/host/src/components/AppBar/SectionIcon.tsx:60]: iffy, and possibly not needed - having a tooltip on the icon. I think it's helpful to explain them though, so put the tooltip on.

File: packages/host/src/components/AppDrawer/AppDrawer.tsx [hunk 4]
```diff
@@ -207,6 +209,11 @@ export const AppDrawer: React.FC = () => {
             icon={<RadioIcon fontSize="small" color="primary" />}
             text={t('sync')}
           />
+          <ExternalNavLink
+            to={ExternalURL.PublicDocs}
+            icon={<BookIcon fontSize="small" color="primary" />}
```
Comment (mark-prins) [inline packages/host/src/components/AppDrawer/AppDrawer.tsx:214]: thinking that if/when we do have specific docs links for specific pages then we can manage that in a number of different ways. That was part of the reason for having the docs link in the AppBarButtons - but I like the button better tucked away in the left nav

Comment (joshxg) [reply packages/host/src/components/AppDrawer/AppDrawer.tsx:214]: Yup!

**Review by joshxg — APPROVED** [review]
This is awesome! Thanks. Think it looks good and nice to have everything consolidated in a single place.

File: packages/common/src/ui/components/navigation/ExternalNavLink.tsx (hunk 1 above)
Comment (mark-prins) [reply packages/common/src/ui/components/navigation/ExternalNavLink.tsx:1]: good point! thanks

**Pull request merged**